src/frontend/src/pages/myfeeds.js

```diff
@@ -14,6 +14,28 @@ const useStyles = makeStyles(theme => ({
 
 export default function MyFeeds() {
   const classes = useStyles();
+  const [feedAuthor, setFeedAuthor] = useState('');
+  const [feedUrl, setFeedUrl] = useState('');
+
+  function handleAuthorChange(author) {
+    setFeedAuthor(author);
+  }
+
+  function handleUrlChange(url) {
+    setFeedUrl(url);
+  }
+
+  function addFeed(author, url) {
+    const http = new XMLHttpRequest();
+    // if testing local dev use the first one, otherwise please use the second apiUrl
+    const apiUrl = 'http://localhost:3000/feeds/';
+    // const apiUrl = 'dev.telescope.cdot.systems/feeds/'
+    const params = `author=${author}&url=${url}`;
+    http.open('POST', apiUrl, true);
+    http.setRequestHeader('Content-type', 'application/x-www-form-urlencoded');
+    http.send(params);
+  }
```

Comment:
Several other frontend components (e.g. [Layout](https://github.com/Seneca-CDOT/telescope/blob/master/src/frontend/src/components/Layout/Layout.js#L57), [Login](https://github.com/Seneca-CDOT/telescope/blob/master/src/frontend/src/components/Login/Login.jsx#L22)) use the [`fetch` API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) to handle HTTP requests. Perhaps for the sake of consistency, we could refactor this function to do the same?